### DISCUSSION TITLE
SSL Issue

### DISCUSSION THREAD

**Discussion by NotGovernor** [NuSkooler/enigma-bbs#376] [Q&A] [answered]
Hey,

Not sure why ssl connectivity isnt working.

- New install on ubuntu server
- ran the install script (success)
- edited config file
- tested via telnet with syncterm (success)
- generated PK using command in config file (success)
```
openssl genpkey -algorithm RSA -pkeyopt rsa_keygen_bits:2048 -pkeyopt rsa_keygen_pubexp:65537 | openssl rsa -out ./config/security/ssh_private_key.pem -aes128
```
- edited config file, specified port, password, set enabled to true
- restarted server (success)
- synterm wont connect when set to ssl with the correct port. "Connect error (0)!"

What am I missing?

Thanks in advance for any assistance. <3

**Comment by NotGovernor** [comment, marked as answer]
You can close this -- I dont understand why but its working now. I just restarted the server an extra time. Sorry for the trouble.

**Comment by NuSkooler** [comment]
@NotGovernor No problem, glad it's working for you!
